config/default.json

```diff
@@ -0,0 +1,52 @@
+{
+    "robotYarpPortPrefix": "/icubSim",
+    "telemVizServer": {
+        "host": "localhost",
+        "port": 8081
+    },
+    "consoleServer": {
+        "host": "localhost",
+        "port": 3000
+    },
+    "openmctStaticServer": {
+        "host": "localhost",
+        "port": 8080
+    },
+    "portInConfig": {
+        "sens.imu": {
+            "yarpName": "${this.robotYarpPortPrefix}/inertial",
+            "localName": "/yarpjs/inertial:i",
```

Comment:
Could it make sense to move this `/yarpjs` prefix to a single variable (like `this.robotYarpPortPrefix`) so that it is easy to change prefix if one wants to launch to yarp-openmct integrations as part of the same YARP network?

Reply:
Sorry if I misunderstood your point, but it actually already works that way. `${this.robotYarpPortPrefix}` points to the value of the field `robotYarpPortPrefix` at the root of the JSON structure. This actually points to the structure object itself (the root).
This is not a native feature of JSON, as the JSON format only supports strings and numbers, but the string interpolation is evaluated in `config/processedDefault.js`.

Reply:
Yes, I was wondering if we could write as well:
~~~
        "sens.imu": {
            "yarpName": "${this.robotYarpPortPrefix}/inertial",
            "localName": "${this.localYarpPortPrefix}/inertial:i",
~~~
By appropriately defining `localYarpPortPrefix` at the top of the json.

Reply:
ah sorry, got it, you were talking about the local. Yes, it is trivial, we can create any variable we want, the same way we defined `${this.robotYarpPortPrefix}`. Good point, I'll do the change.

Reply:
Fixed in #75 .

Reply:
Thanks!